src/bin/cp2cp.rs

```diff
@@ -43,27 +47,46 @@ fn main() {
     }
     logger.init().unwrap();
 
-    //log::info!("=====================================================");
+    // log::info!("=====================================================");
     //log::info!("{} starting up!", std::module_path!());
     //log::info!("=====================================================");
     //log::info!("Verbosity levels: {}", verbosity_string);
 
+    if opts.chainpack_rpc_block {
+        opts.indent = None;
+        opts.chainpack_output = false;
+        opts.cpon_input = false;
+    }
+
     let mut reader: Box<dyn BufRead> = match opts.file {
         None => Box::new(BufReader::new(io::stdin())),
         Some(filename) => Box::new(BufReader::new(fs::File::open(filename).unwrap()))
     };
 
+    let mut chainpack_rpc_block_header = None;
     let res = if opts.cpon_input {
         let mut rd = CponReader::new(&mut reader);
         rd.read()
     } else {
         let mut rd = ChainPackReader::new(&mut reader);
+        if opts.chainpack_rpc_block {
+            let length = rd.read_uint_data().unwrap();
+            let proto = rd.read().unwrap().as_int();
+            chainpack_rpc_block_header = Some((length, proto));
+        }
         rd.read()
     };
     let rv = match res {
         Err(e) => {
-            eprintln!("Parse input error: {:?}", e);
-            process::exit(1);
+            match e.reason {
+                ReadErrorReason::UnexpectedEndOfStream => {
+                    process::exit(2);
```

Comment:
What is the meaning of exit code `2` (and `1` in the next match arm) here? Constants would be better.

Reply:
agree